lib/battle/CBattleInfoCallback.cpp

```diff
@@ -164,8 +157,89 @@ ESpellCastProblem::ESpellCastProblem CBattleInfoCallback::battleCanCastSpell(con
 	return ESpellCastProblem::OK;
 }
 
+struct Point

```

Comment:
can't believe it's not defined somewhere

Reply:
It is defined. In client. With dependencies on SDL. But this code needs to be in library class, without SDL dependency.

Reply:
Will try to separate our Point & Rect from SDL, but in another PR. Definitely needed, but outside of PR scope